docs/proposals/localDisk_volume.md

```diff
@@ -0,0 +1,309 @@
+# localDisk volume
+
+## Abstract
+
+This is a proposal for implementing a new volume type, localDisk, which will support,
+
+ 1. the Node has multiple physical disks and pod requires 1 ~ N disks as its storage volumes.
+ 2. the Node has lvm VG and pods allocate specific size LVs from this VG.

```

Comment:
this proposal needs a terminology section that defines the linux specific terms like logical volume and volume group up top. 

What creates the VG?